File: src/main/java/org/jukeboxmc/network/handler/RequestChunkRadiusHandler.java
package org.jukeboxmc.network.handler;

import org.cloudburstmc.protocol.bedrock.packet.RequestChunkRadiusPacket;
import org.jukeboxmc.Server;
import org.jukeboxmc.player.Player;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class RequestChunkRadiusHandler implements PacketHandler<RequestChunkRadiusPacket> {

    @Override
    public void handle( RequestChunkRadiusPacket packet, Server server, Player player ) {
        player.setChunkRadius( packet.getRadius() );
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/network/handler/RequestChunkRadiusHandler.kt
package org.jukeboxmc.server.network.handler

import org.cloudburstmc.protocol.bedrock.packet.RequestChunkRadiusPacket
import org.jukeboxmc.server.JukeboxServer
import org.jukeboxmc.server.player.JukeboxPlayer

class RequestChunkRadiusHandler : PacketHandler<RequestChunkRadiusPacket> {

    override fun handle(packet: RequestChunkRadiusPacket, server: JukeboxServer, player: JukeboxPlayer) {
        player.getPlayerChunkManager().setChunkRadius(packet.radius)
    }
}